core/domain/feedback_services.py

```diff
@@ -422,112 +417,108 @@ def get_thread_summaries(user_id, thread_ids):
 
     Args:
         user_id: str. The id of the user.
-        thread_ids: str. The ids of the threads for which we have to fetch the
-            summaries.
+        thread_ids: list(str). The ids of the threads for which we have to fetch
+            the summaries.
 
     Returns:
-        list(dict). A list of dictionaries containing the summaries of the
-            threads given to it. Each dict has the following keys:
-            - 'status': str. The status of the thread.
-            - 'original_author_id': str. The id of the original author of the
-                thread.
-            - 'last_updated': datetime.datetime. When was the thread last
-                updated.
-            - 'last_message_text': str. The text of the last message.
-            - 'total_message_count': int. The total number of messages in the
-                thread.
-            - 'last_message_is_read': boolean. Whether the last message is read
-                by the user.
-            - 'second_last_message_is_read': boolean. Whether the second last
-                message is read by the user,
-            - 'author_last_message': str. The name of the author of the last
-                message.
-            - 'author_second_last_message': str. The name of the author of the
-                second last message.
-            - 'exploration_title': str. The title of the exploration to which
-                exploration belongs.
-        int. The number of threads not read by the user.
-    """
-    feedback_thread_user_model_ids = [
+        tuple(thread_summaries, number_of_unread_threads), where:
+            thread_summaries: list(dict). A list of dictionaries containing the
```

Comment:
Should we be returning a list of domain objects instead?

Reply:
Done.